roles/network_plugin/flannel/templates/cni-flannel.yml.j2

```diff
@@ -41,25 +40,22 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: kube-flannel
-  namespace: "kube-system"
+  namespace: kube-system
   labels:
     tier: node
-    k8s-app: flannel
+    app: flannel
 spec:
   selector:
     matchLabels:
-      tier: node
-      k8s-app: flannel
+      app: flannel
   template:
     metadata:
       labels:
         tier: node
-        k8s-app: flannel
+        app: flannel
     spec:
       priorityClassName: system-node-critical
       serviceAccountName: flannel
-      nodeSelector:
```

Comment:
I realize I'm late to the party (this was already merged), but the nodeselector is needed for those who might be running windows nodes in their clusters.

Reply:
Hum wasn't sure that latest k8s version and flannel weren't compatible with windows.
But if you're sure let's add it back then :)